bin/queuePopulator.js

```diff
@@ -3,13 +3,13 @@ const schedule = require('node-schedule');
 
 const werelogs = require('werelogs');
 
-const config = require('../../../conf/Config');
+const config = require('../conf/Config');
 const zkConfig = config.zookeeper;
-const repConfig = config.extensions.replication;
-const sourceConfig = config.extensions.replication.source;
-const QueuePopulator = require('./QueuePopulator');
+const extConfigs = config.extensions;
+const qpConfig = config.queuePopulator;
+const QueuePopulator = require('../lib/queuePopulator/QueuePopulator');
 
-const log = new werelogs.Logger('Backbeat:Replication:task');
+const log = new werelogs.Logger('Backbeat:QueuePopulator');
```

Comment:
I though it would add more confusion than benefits, being more an internal split in the code structure than a really different module, so I removed the `:task` on purpose to have the same log prefix than in the `QueuePopulator` class. We could also consider passing this log object to the `QueuePopulator` constructor.